src/app/components/layout/navbar/navbar.component.html

```diff
@@ -192,7 +192,7 @@
                 </a>
               </div>
               <div class="dropdown-menu" id="dropdown-menu" role="menu">
-                <div class="dropdown-content">
+                <div *ngIf="servicesRecord.length !== 0" class="dropdown-content">
```

Comment:
do we need this here?  

Are we missing on  `<ng-container *ngIf="!showSpinner">`